courscala/src/main/scala/org/coursera/common/stringkey/StringKeyFormat.scala

```diff
@@ -207,3 +229,84 @@ private case object UuidFormat extends StringKeyFormat[UUID] {
   }
 
 }
+
+/**
+ * Modified from [[TupleFormats.Tuple2Format]].
+ */
+private case class SeqFormat[T](separator: String)(implicit format: StringKeyFormat[T])
```

Comment:
These are both private right now. We'd need to add appropriate methods in `CommonStringKeyFormats` to expose them, I think.

I originally left them out because of their undefined behavior with empty key representations. Since we don't check (with an exception) for that, these encodings are somewhat unsafe. If we still need them in Naptime, could they live in Naptime's router (with restricted visibility) instead of here?